default/scripting/species/common/population.macros

```diff
@@ -225,6 +225,19 @@ GASEOUS_BONUS
             effects = SetTargetPopulation value = Value - 0.5*abs(Value)
 '''
 
+MINDLESS_BONUS
+'''     EffectsGroup
+            scope = Source
+            activation = And [
+                Planet
+                Not Planet type = Asteroids
+                HasTag name = "MINDLESS_SPECIES"
```

Comment:
Notice that this population malus is not affecting Exobots on asteroids, and so that part of the game balance is not changed at all.

Reply:
This isn't working. Exobots are not getting the -50% population. Something in the activation conditions is wrong.
@geoffthemedio @agrrr3 help, please.

Reply:
Solved. I didn't add the new effect to BASIC_POPULATION.